src/components/layout/TopIconButton.jsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import './TopIconButton.less';
 
-const TopIconButton = ({ name, icon, onActiveTab = () => {}, isActive = false }) => (
+const TopIconButton = ({ name, icon, onActiveTab = () => {}, isActive = false, tabIndex = '0' }) => (
   <div
     className={isActive ? 'top-icon-button button-top-active body-typo' : 'top-icon-button body-typo'}
     onClick={onActiveTab}
     onKeyDown={onActiveTab}
     role='button'
-    tabIndex={0}
+    tabIndex={tabIndex}
```

Comment:
why this change (and all the changes above with setting this value to `-1`)?

Reply:
This line you commented on -- it's propagating the new tabIndex property I added into the div. The new property I added sets this value to -1 because the tab order is going to be using the link rather than the button. Before my change, tabbing the top bar nav buttons had two tabs per button. Now there's only one tap per button. If you test out my new tab order, it works better :D

Reply:
The "tab order" refers to the user experience while navigating the page using just tab, shift+tab, and enter.

Reply:
i wonder if we should default it to `-1`, but minor